app/store/middlewares/helpers.js

```diff
@@ -48,6 +55,13 @@ export function cleanUpState(payload, keepCurrent = false) {
 
     const postIdsToKeep = [];
 
+    // Keep the last 60 threads in each team
+    nextEntities.threads = {
+        ...nextEntities.threads,
+        ...cleanUpThreadsInTeam(payload.entities.threads?.threads, payload.entities.threads?.threadsInTeam),
+    };
```

Comment:
what happens if I send the app to the background and I'm in the thread screen and I've scrolled past the 60 threads mark?